### ISSUE TITLE
Can't install on Joomla 4

### ISSUE THREAD

**Issue by dgjoom** [nikosdion/combinator#1]
Just try to Install the plugin on Joomla 4 at local host and it look that can't install it.
Try at Local Host, Joomla 4.0.3
Iv got 2 warning message :
- JInstaller: :Install: Can't find Joomla XML setup file.
- Unable to detect manifest file.

**Comment by nikosdion** [comment]
This plugin is not compatible with Joomla 4.

Joomla 4 has proper support for HTTP/2 Push, deferred loading of JavaScript files, service pre-compressed (Gzip or Brotli) CSS/JS files and serving JavaScript modules. These are **FAR MORE RELIABLE** than concatenating JS and CSS files... and result in better performance, too! It makes sense for developers to use these features instead.

Modern browsers also offer a much better HTTP/2 Push support. Instead of stupidly accepting every single file the server wants to push they can instead tell the server which of the pushed files they need to be sent.

In short, there is no reason for me to maintain this plugin anymore. Joomla 4's improvements lead to much faster, much more stable sites without the need of iffy solutions like combining JS and CSS files.
